lib/warder/coffee_lint_runner.rb

```diff
@@ -0,0 +1,60 @@
+require 'coffeelint'
+
+module Warder
+  # responsible for run coffeescript style guide validation
+  class CoffeeLintRunner < Runner
+    CLI_OPTION = 'g'
+    CLI_FULL_OPTION = 'coffee-lint'
+    DESCRIPTION = 'Run coffeescript style guide validation'
+    COMMAND_NAME = 'coffeelint'
+    FAILURE_REGEXP = /(?<issues>(WARN|ERROR))/
+
+    def run_command
+      @stdout.puts(exec_msg) unless quiet?
+
+      # # print single line results
+      # line = lint_results.map do |filename, result|
+      #   if result.empty?
+      #     '.'
+      #   else
+      #     if result.any? { |res| res['level'] == 'error' }
+      #       'E'
+      #     else # 'warn'
+      #       'W'
+      #     end
+      #   end
+      # end.join
+      # yield(line)
+
+      # print error messages
+      lint_results.each do |filename, result|
+        unless result.empty?
+          yield("\n#{filename}")
+          result.each do |res|
+            yield("#{res['level'].upcase}: ##{res['lineNumber']}: #{[res['message'], res['context']].join('. ')}.")
+          end
+        end
+      end
+
+      @stdout.puts(stats_msg) if stats?
+    end
+
+    def lint_results
+      return @lint_results if defined?(@lint_results)
+      lint_results = {}
+      @options.files.split(' ').each do |filename|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.